### ISSUE TITLE
Zen mode reviving

### ISSUE THREAD

**Issue by ONYX4407** [mb1986/rm-hacks#344]
I think not only me would greatly appreciate the revival of Zen mode from the ddvk hack. 

I belive it was the most elegant and simple mode for no distractions.

Is it possible to integrate this into your hack please?

Thanks a lot!

**Comment by PepikVaio** [comment]
What exactly was the task of this mod?
As far as I know, only the toolbar was completely hidden even by the ball at the top.
Or did it have something more?

**Comment by ilbonte** [comment]
I think it was a single tap on the most left or right side to turn pages (instead of swiping)

**Comment by Ram-222** [comment]
I think that may have been the separate reading mode. But I'd also like that again. Was nicer for reading books to just tap to swipe, but in particular was really handy when using the RM to read music whilst playing, as much quicker to tap than swipe when need fingers back on the keys!

**Comment by ONYX4407** [comment]
@ilbonte @Ram-222 No, I meant the mode, where you can see onny one icon as is shown in QuickTool panel (after. hiding default panel) where you can change stroke size, colore, swicht last pen and switch to eraser like is shown on the picture below. 

Also "double tap" to redo isnt much freindy against to to swipe to left on the middle-top of screen, same as on the picture.

<img width="957" alt="image" src="https://github.com/mb1986/rm-hacks/assets/8388757/e11d5467-4052-43d7-b76d-ba91c753e428">

**Comment by atngames** [comment]
Ok so this is feature that has been requested many times, with many different forms.

I think what we need there is a discussion on what would be rmhacks workflow overall. 

A reading / Zen / low energy mode can be implemented, for sure, but had to satisfy both ddvk's users and other requests. 

I'll think about it soon

**Comment by PepikVaio** [comment]
@ONYX4407 I still don't understand - I guess I'm stupid - what exactly zen mode does. 
You mean like this????

![image](https://github.com/mb1986/rm-hacks/assets/36114753/f9f9e0a4-a132-42bd-8ee2-39a8e954896a)
